README.md

```diff
@@ -34,6 +33,15 @@ Commonly used Gulp tasks shared across projects.
 - `gulp fractal:start`: Start Fractal server (if config path exists in `package.json`)
 - `gulp fractal:build`: Build static version of Fractal project
 
+## Options for js/css tasks
+
+`--minify` | Render additional minified files, suffixed with `.min`. Will be `true` in production builds.
+`--sourcemaps` | Render sourcemaps.
+`--fail-after-error` | See the 'Continuous Integration' section for details.
+`--no-concat` | By default, assets are compiled into a single file. Use this flag to keep files invididual.
```

Comment:
It seems like most projects that are using `gulp-task-collection` are disabling concatenation.

If these changes are released as a 3.0, does it make sense to change the default concat to false?

Reply:
Changing the default to be disabled sounds sensible to me 👍

Reply:
concat is now `false` by default